src/pages/en/interventions/44-makerspace-bootcamp.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: "@layouts/InterventionSingleView.astro"
+title: Makerspace BootCamp for School Teams 
+subtitle:
+date: October 2nd - 14th, 2023
+location: Kyiv
+locationLngLat: [30.523333, 50.450001] 
+tags: 
+  - Education, Replication
+operator: Ostriv Platform
+car: Tolocar 5
+images:
+  - images/interventions/makerspace-bootcamp-1.jpg  
+  - images/interventions/makerspace-bootcamp-2.jpg
+  - images/interventions/makerspace-bootcamp-3.jpg 
+  - images/interventions/makerspace-bootcamp-4.jpg
+  - images/interventions/makerspace-bootcamp-5.jpg
+  - images/interventions/makerspace-bootcamp-6.jpg
+  - images/interventions/makerspace-bootcamp-7.jpg
+  - images/interventions/makerspace-bootcamp-7.jpg
+  - images/interventions/makerspace-bootcamp-7.jpg
+  - images/interventions/makerspace-bootcamp-7.jpg
```

Comment:
@Jacky-Bertlich Are you sure, you want to use the same image four times here? I guess you actually want to use the images ending in `7`, `8` and `9`, right?

Reply:
Oh yes, that should be numbered from /makerspace-bootcamp-1 to .. 10 in a row! three times -7 is a error, yes.